examples/stable_diffusion_xl/inference_lcm.md

```diff
@@ -0,0 +1,74 @@
+# LCM Sampler for Stable Diffusion XL (SDXL)
+
+[Latent Consistency Models: Synthesizing High-Resolution Images with Few-Step Inference](https://arxiv.org/abs/2310.04378)
+
+Latent Consistency Models (LCM) enable quality image generation in typically 2-4 steps making it possible to use diffusion models in almost real-time settings.
+
+## Dependency
+
+- mindspore 2.2
+- openmpi 4.0.3 (for distributed mode)
+
+To install the dependency, please run
+
+```shell
+pip install -r requirements.txt
+```
+
+## Preparation
+
+### Convert Pretrained Checkpoint
+
+We provide a script for converting pre-trained weight from `.safetensors` to `.ckpt` in `tools/model_conversion/convert_weight.py`.
+
+step1. Download the [Official](https://github.com/Stability-AI/generative-models) pre-train weights [text_encoder](https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/tree/main/text_encoder), [text_encoder_2](https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/tree/main/text_encoder_2), [vae](https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/tree/main/vae) and [
+lcm-sdxl](https://huggingface.co/latent-consistency/lcm-sdxl/tree/main) from huggingface.
+
+step2. Convert a HF Diffusers saved pipeline to a Stable Diffusion checkpoint.
+
+```shell
+cd tools/model_conversion
+
+python convert_diffusers_to_original_sdxl.py \
+  --model_path /PATH TO THE MODEL TO CONVERT \
+  --checkpoint_path /PATH TO THE OUTPUT MODEL/sd_xl_base_1.0.safetensors \
```

Comment:
what's `/PATH TO THE MODEL TO CONVERT` & `/PATH TO THE OUTPUT MODEL`?

Reply:
BTW, do NOT use space as seperator

Reply:
--model_path (Path to the model to convert): the file path of saving the official pre-train weights text_encoder, text_encoder_2, vae and lcm-sdxl from huggingface
--checkpoint_path (Path to the output model): the file path of saving sd_xl_base_1.0.safetensors (SDXL model path)